examples/custom_3d_three-d/Cargo.toml

```diff
@@ -12,4 +12,4 @@ publish = false
 eframe = { path = "../../eframe", features = ["glow"] }
 egui_glow = { path = "../../egui_glow" }
 glow = "0.11"
-three-d = { version = "0.11", default-features = false }
+three-d = { version = "0.12", default-features = false }
```

Comment:
I think we should use `git = "https://github.com/asny/three-d.git", rev = "7ac4f3e1e14335290e505a5799a0b88717474678"` here to make it easier for others to copy-paste this example without getting surprise errors!